acceptance/services_test.go

```diff
@@ -150,7 +150,35 @@ var _ = Describe("Services", LService, func() {
 		})
 	})
 
-	Describe("Show", func() {
+	Describe("Show with credentials", func() {
+		var service string
+
+		BeforeEach(func() {
+			service = catalog.NewServiceName()
+
+			By("create it")
+			out, err := env.Epinio("", "service", "create", "mysql-dev", service, "--wait")
+			Expect(err).ToNot(HaveOccurred(), out)
+		})
+
+		It("shows a service which has credentials", func() {
+			By("show it")
+			out, err := env.Epinio("", "service", "show", service)
+			Expect(err).ToNot(HaveOccurred(), out)
+			Expect(out).To(ContainSubstring("Showing Service"))
+			Expect(out).To(ContainSubstring("No settings"))
+			Expect(out).To(ContainSubstring("Credentials:"))
+			Expect(out).To(
+				HaveATable(
+					WithHeaders("KEY", "VALUE"),
```

Comment:
I expect here to see also the username, not only the password; shouldn't the username key/value checked here as well?

Reply:
The change shows all the keys and their values in all the associated K secrets of the service.
The keys in the mysql service secrets do not provide any kind of user information.
I suspect that the helm chart (a bitnami one) uses a fixed username.